### PR TITLE
Fix source comment typos and source typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S *.po,*.xpm,./lib-src -L ans,ba,clen,fiter,hashi,imagin,inout,levl,nin,ontext,optionaly,pard,parm,parms,pevent,pixelx,screem,siz,sord,strack,thisy,toke`

### CODE REVIEW

File: plug-ins/sample-data-import.ny [hunk 1]
```diff
@@ -40,11 +40,11 @@ $control bad-data (_ "Invalid data handling") choice (("ThrowError" (_ "Throw Er
         t)))
 
 ;; ':new' creates a new class 'streamreader'
-;; 'filestream' and 'chanel' are its instance variables.
+;; 'filestream' and 'channel' are its instance variables.
 ;; (every object of class 'streamreader' has its own
 ;; copy of these variables)
 (setq streamreader
-  (send class :new '(filestream chanel)))
+  (send class :new '(filestream channel)))
```
Comment (luzpaz) [inline plug-ins/sample-data-import.ny:47]: Is this OK?

Comment (LWinterberg) [reply plug-ins/sample-data-import.ny:47]: LGTM - @SteveDaulton confirm?

Comment (Paul-Licameli) [reply plug-ins/sample-data-import.ny:47]: > LGTM - @SteveDaulton confirm?

I can tell you that the fix is incomplete and incorrect, unless you change all "chanel" to "channel" and there is in fact more than one occurrence.

Comment (luzpaz) [reply plug-ins/sample-data-import.ny:47]: @Paul-Licameli JFTR, i did a sourcecode-wide search for 'chanel' and came up with nothing. What am I missing?

Comment (Paul-Licameli) [reply plug-ins/sample-data-import.ny:47]: You are right, I was confused

**Review by LWinterberg — APPROVED** [review]

File: plug-ins/sample-data-import.ny (hunk 1 above)
Comment (SteveDaulton) [reply plug-ins/sample-data-import.ny:47]: Looks OK to me.

**Pull request merged**